bootstrap-salt.sh

```diff
@@ -2454,10 +2459,10 @@ install_ubuntu_stable_deps() {
         # Versions starting with 2015.5.6, 2015.8.1 and 2016.3.0 are hosted at repo.saltstack.com
         if [ "$(echo "$STABLE_REV" | egrep '^(2015\.5|2015\.8|2016\.3|2016\.11|latest|archive\/)')" != "" ]; then
             # Workaround for latest non-LTS ubuntu
-            if [ "$DISTRO_VERSION" = "16.10" ]; then
+            if [ "$DISTRO_VERSION" = "16.10" ] || [ "$DISTRO_VERSION" = "17.04" ]; then
```

Comment:
I guess maybe it would be better to verify that `[ "$DISTRO_MAJOR_VERSION" -gt 16 ]` as above?
It will "automagically" enable support for future `17.10` release.